### PR TITLE
KIALI-1463 UX Feedback for Graph

### PR DESCRIPTION
**Pull request by jshaughn**
This work is to apply the design review in https://github.com/kiali/kiali-design/pull/42.

### CODE REVIEW

**Comment by abonas** [comment]
is the commit message describing what are the UX changes? I didn't see, perhaps I missed it. the main comment referring to the issue in the design repo will be "lost" when this is merged

**Comment by jshaughn** [comment]
@abonas Don't worry, this is still DNM, eventually I'll squash and update the final commit message.  Did you have any comments on the screenshots I posted in the design issue?

**Comment by jshaughn** [comment]
@cshinn Not asking to review code, just added you as a reviewer for the design.  You can see the screenshots in your design PR: https://github.com/kiali/kiali-design/pull/42.

**Comment by cshinn** [comment]
@jshaughn whoops! just realized that I commented on the design thread rather than here (and tagged the wrong person initially to boot). I'll duplicate that comment below:

Looks good @jshaughn! I've got a couple questions and things that we could try:

- I really like the badges before the label, and I think the color change works well, but I don't think the text needs to be bold also. I think it makes it look a bit muddy.
- The shapes could maybe be a little bit larger relative to the text. They may be a bit tricky to click on and at this size an octagon and a circle look almost identical.
    - What is the font size and shape size that you are using here?
- Zoom tool looks awesome.
- Edge text looks good, maybe a touch more opacity would make it easier to read if they're always going to overlap the edges.

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#5](https://paste.fedoraproject.org/paste/W4bXpAbEmqd6C2fVuNZ11A)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1152]](https://paste.fedoraproject.org/paste/C9W2EJAZNLewewUIuYr8PA)
  * :heavy_check_mark: install-bookinfo [#[574]](https://paste.fedoraproject.org/paste/i6DGktY7yRsKW2XipCUZHg)
  * :red_circle: Kiali-on-Openshift [#[503]](https://paste.fedoraproject.org/paste/KHjS4vydDZa4TDA9nmt0pg)

**Comment by jshaughn** [comment]
Removing the DNM label.  This is ready for merge, we can always iterate with UX.  Note the updated legend and the change to triangle for service nodes.  The triangle is much easier to see at smaller size.  Note also that we may want to consider moving to 'vee' arrowheads, they are less obtrusive and also avoid another use of triangle.

screenshot

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#6](https://paste.fedoraproject.org/paste/QcBzhFA~7j0smv6hkebulg)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1153]](https://paste.fedoraproject.org/paste/3U2HN91CFC0Zv5KSkKNWKw)
  * :heavy_check_mark: install-bookinfo [#[575]](https://paste.fedoraproject.org/paste/AdtCPf5czq92~Jw1RuHBVA)
  * :red_circle: Kiali-on-Openshift [#[504]](https://paste.fedoraproject.org/paste/WkJG3yPylFZvv5LVvJGsHA)

**Comment by cshinn** [comment]
Looks good @jshaughn! Two strange things though:
- The serif font in the legend doesn't match the font anywhere else on the page
- The services as triangles are exactly the same size as the triangles at the ends of the edges. Could be misleading especially in cases of error conditions, like a second very short arrow pointing off in a different direction

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#7](https://paste.fedoraproject.org/paste/VD2Nloun4GNTwbEZjAcUDw)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1154]](https://paste.fedoraproject.org/paste/-2tehqkEwasFhwZ4FS2HEw)
  * :heavy_check_mark: install-bookinfo [#[576]](https://paste.fedoraproject.org/paste/GrmfWx5BY27-B4-EZzKfVg)
  * :red_circle: Kiali-on-Openshift [#[505]](https://paste.fedoraproject.org/paste/ccpHpM76zV-qq4xdgXa~lA)

**Comment by jshaughn** [comment]
added back the DNM label as I address some comments by @cshinn .

**Comment by mwringe** [comment]
Do we want the down arrow to represent that something is in different namespace?

Maybe the fa-external-link or pficon-topology from the patternfly icons:  https://www.patternfly.org/styles/icons/

@cshinn any thoughts here?

**Comment by josejulio** [comment]
> The serif font in the legend doesn't match the font anywhere else on the page

@cshinn 
What font should i use for the legend text? (I updated the legend)

**Comment by cshinn** [comment]
@josejulio OpenSans is probably best

**Comment by mwringe** [comment]
@cshinn And if we didn't want to use that icon, do you have another suggestion? I have heard some feedback that it doesn't really depict that it represents that it exists in another namespace and that if you squint that it might look like something inappropriate.

**Comment by cshinn** [comment]
@mwringe ah, interesting point. I'll look into it and get back to you

**Comment by cshinn** [comment]
@jshaughn that all sounds great, do you have a pic of the new stuff?

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#8](https://paste.fedoraproject.org/paste/xQW9BofGBakfadMy9il2AA)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1156]](https://paste.fedoraproject.org/paste/RWZNFLUqJSLo00Um8BBN~w)
  * :heavy_check_mark: install-bookinfo [#[577]](https://paste.fedoraproject.org/paste/DfJoo2b4Yz4SEv8OTk7Iyw)
  * :red_circle: Kiali-on-Openshift [#[506]](https://paste.fedoraproject.org/paste/0LtxkQC5VjzedtGAfb7J8A)

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#9](https://paste.fedoraproject.org/paste/PQvSRG1bKHg06SVU4hDWZg)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1158]](https://paste.fedoraproject.org/paste/dwOiTfP6RBiRVprzEIzD7g)
  * :heavy_check_mark: install-bookinfo [#[578]](https://paste.fedoraproject.org/paste/qwYhV0Xz3E137vj~aAJIpg)
  * :red_circle: Kiali-on-Openshift [#[507]](https://paste.fedoraproject.org/paste/BfDDNMKX~NjzJErWFGXR5g)

File: src/pages/WorkloadDetails/WorkloadInfo.tsx [hunk 1]
```diff
@@ -114,7 +114,7 @@ class WorkloadInfo extends React.Component<WorkloadInfoProps, WorkloadInfoState>
                       {pods.length > 0 && <WorkloadPods pods={pods} validations={this.props.validations!['pod']} />}
                     </TabPane>
                     <TabPane eventKey={'services'}>
-                      {pods.length > 0 && <WorkloadServices services={services} namespace={this.props.namespace} />}
+                      {services.length > 0 && <WorkloadServices services={services} namespace={this.props.namespace} />}
```
Comment (josejulio) [inline src/pages/WorkloadDetails/WorkloadInfo.tsx]: This is unrelated to this PR, did that other commit got mixed?

Comment (jshaughn) [reply src/pages/WorkloadDetails/WorkloadInfo.tsx]: @josejulio This came from your original legend PR.

Comment (josejulio) [reply src/pages/WorkloadDetails/WorkloadInfo.tsx]: oh right, i think is related to this commit  https://github.com/kiali/kiali-ui/commit/db3c7b3e3f646c2b98ea21beb6e3d4f04e1bb7ee

**Comment by jshaughn** [comment]
@cshinn an image:

screenshot

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#10](https://paste.fedoraproject.org/paste/F5LSVbCmvUqyNl4OFM8l-w)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1159]](https://paste.fedoraproject.org/paste/VpahmBxAERqiwdeNsenRWA)
  * :heavy_check_mark: install-bookinfo [#[580]](https://paste.fedoraproject.org/paste/XWXZ~h7gDFu7bVY3bYn~LA)
  * :red_circle: Kiali-on-Openshift [#[511]](https://paste.fedoraproject.org/paste/EpVDJW2sxaStIwUUJOT0Og)
